signingscript/src/signingscript/utils.py

```diff
@@ -92,11 +92,15 @@ def _load_scoped_configs(filename, cls, name):
     with open(filename) as f:
         raw_cfg = json.load(f)
 
-    cfg = {}
+    scope_configs = {}
+    # TODO: We should refactor how Autograph data is loaded to be key-pair instead of list
```

Comment:
Yea

Reply:
If we refactored that you would not need the if else on the `cls` instantiation?